.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "perf_analyzer Dev Container",
+  "image": "gitlab-master.nvidia.com:5005/dl/dgx/tritonserver:master-py3-devel",
```

Comment:
```suggestion
  "image": "gitlab-master.nvidia.com:5005/dl/dgx/tritonserver:master-py3-base",
```

Reply:
When did we start using the base container for PA build? I was using devel until now 😅

Reply:
Ask @matthewkotila - originally I had the -devel too :D

Reply:
You might have different needs than me, but I use base because it still has triton and is smaller than devel (faster to pull).

Reply:
No particular needs. I was mainly using it because our onboarding doc uses it. Good to know since I would like to keep things more minimal 👍

Reply:
![Uploading gif.gif…]()